CONTRIBUTING.rst

```diff
@@ -0,0 +1,153 @@
+How to contribute to Flask-SQLAlchemy
+=====================================
+
+Thank you for considering contributing to Flask-SQLAlchemy!
+
+Support questions
+-----------------
+
+Please, don't use the issue tracker for this. Use one of the following
+resources for questions about your own code:
+
+* The IRC channel ``#pocoo`` on FreeNode.
+* The IRC channel ``#python`` on FreeNode for more general questions.
+* The mailing list flask@python.org for long term discussion or larger issues.
+* Ask on `Stack Overflow`_. Search with Google first using:
+  ``site:stackoverflow.com flask-sqlalchemy {search term, exception message, etc.}``
+
+.. _Stack Overflow: https://stackoverflow.com/questions/tagged/flask-sqlalchemy?sort=linked
+
+Reporting issues
+----------------
+
+- Describe what you expected to happen.
+- If possible, include a `minimal, complete, and verifiable example`_ to help
+  us identify the issue. This also helps check that the issue is not with your
+  own code.
+- Describe what actually happened. Include the full traceback if there was an
+  exception.
+- List your Python, Flask, and Werkzeug versions. If possible, check if this
+  issue is already fixed in the repository.
+
+.. _minimal, complete, and verifiable example: https://stackoverflow.com/help/mcve
+
+Submitting patches
```

Comment:
Should we encourage people to open issues for discussion before jumping in with a PR?